pages/Graph/Overview.md

```diff
@@ -16,13 +21,81 @@ This section should also cover how our graph representation could be applied to
 | 0.1     | Draft |
 
 ## Purpose
-
-TBD
+1. Describe the form of Graph Message posted to the blockchain
+1. Explain how to retrieve and resolve the graph
+1. Facilitate use of SDK and interpretation of graph-data
 
 ## Assumptions
+- All graph messages will be of type `GraphChange`.
+- All `GraphChange` messages will be added to batch files for an announcement on block chain
+
+##Terminology
+
+#### Friendship
+
+There is no concept of "friendship" within the DSNP social network graph. 
+Friendship requires a mutual acknowledgement between 2 different DSNP User Identities. 
+Friendship can be thought as "mutual following" - where 2 DSNP User identities are following each other. 
+
+#### Follow
+
+A "follow" is the act of publicly following a user (referenced as a (DSNP User Identity)[/Identity/Overview]) which results in adding this DSNP User Id to a user's social graph.
+
+#### Unfollow
+
+An "unfollow" is the act of publicly unfollowing a user (referenced as a (DSNP User Identity)[/Identity/Overview]) which results in the removal of this DSNP User Id from a user's social graph.
+
+### Graph
+
+The collection of DSNP User identities that a given user is following via the DSNP protocol. 
+
+#### Public
+Publicly following or unfollowing a DSNP User Id implies that anyone looking for these follow
+or unfollow events on the blockchain will be able to determine who the follower(DSNP User Id 
+doing the following) and followee(DSNP User Id being followed) are for a given event.
+
+## Graph Change Announcement Format
+Anytime a user "follows" or "unfollows" someone - a graph change event is created.
+A graph change will contain the following information: `fromId`, `changeType`, `objectId`, `nonce`. See [Messages Overview](/Messages/Overview) for more information on these fields.
+
+## Graph Storage
+All graph change events are signed and added to a [batch file](/Batches/Overview).
```

Comment:
event --> message?

Reply:
perhaps mention the graph DSNPType here.

Reply:
I think as above maybe announcement is better here? just like above. More in line with the spec? - https://spec.dsnp.org/Messages/Overview But good call out. 
And I link to the overview types just above - so the intent was to not repeat things here.